tests/lib/rules/prefer-const.js

```diff
@@ -117,17 +117,45 @@ ruleTester.run("prefer-const", rule, {
             parser: fixtureParser("babel-eslint5/destructuring-object-spread")
         },
 
-        // https://github.com/eslint/eslint/issues/8308
+        // // https://github.com/eslint/eslint/issues/8308
         {
             code: "let predicate; [typeNode.returnType, predicate] = foo();",
             parserOptions: { ecmaVersion: 2018 }
         },
+        {
+            code: "let predicate; [typeNode.returnType,, predicate] = foo();",
```

Comment:
Double comma might be a typo? (Or are you intentionally testing array destructuring skips here?)

Reply:
Intentionally testing a missing spot here.